### PR TITLE
Add usage documentation about `skip` option

### PR DESCRIPTION
**Pull request by brunoocasali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,21 @@ then in your component or controller 's actions:
   }
 ```
 
+### Skipping real library download
+
+Sometimes you want to have the behavior of a reCAPTCHA but without the burden of injecting the downloaded library into your web page. This is particularly useful during tests.
```
Comment (brunoocasali) [inline README.md:118]: @sinankeskin feel free to suggest improvements :)

File: tests/integration/components/g-recaptcha-test.js [hunk 2]
```diff
@@ -78,13 +78,12 @@ module('GRecaptchaComponent', function (hooks) {
 
     test('invokes callbacks normally', async function (assert) {
       // since the callbacks are controlled by the Google code, the only way to force them is using
-      // the grecaptcha.execute() function + size=invisible option.
+      // the grecaptcha.execute() function option.
```
Comment (brunoocasali) [inline tests/integration/components/g-recaptcha-test.js:81]: This is not required anymore (I forgot to remove it in the last PR)

File: README.md (hunk 1 above)
Comment (brunoocasali) [reply README.md:118]: btw: I'm not sure if needs a CHANGELOG entry about that... 🤔

**Review by sinankeskin — APPROVED** [review]
Looks good to me. Thanks @brunoocasali

File: README.md (hunk 1 above)
Comment (sinankeskin) [reply README.md:118]: I don't think so.

**Pull request merged**